docs/contributors/dev_hacking.rst

```diff
@@ -193,7 +193,7 @@ Run the server
 
 Run the server like this::
 
-    $ ./manage.py runserver --traceback
+    $ ./manage.py runserver 0.0.0.0:8000 --traceback # 0.0.0.0 allows access to the local Django dev server from other machines on the network.
```

Comment:
Two things:

1. this only applies in certain situations, so I'd mention that as a note and not change this example
2. instead of putting in a bash comment, it's better to have explanation text as its own paragraph.

This instead of this change, I'd do something like this after the "Then point your browser ..." paragraph:

```
.. note::

   Depending on how and where you're running runserver, you might need to specify
   the ip address and port to bind to. You can provide those as an argument to
   the runserver command. For example::

       $ ./manage.py runserver 0.0.0.0:8000

   will have runserver listen on all ip addresses at port 8000.
```